Makefile.am

```diff
@@ -126,8 +126,6 @@ src_libzmq_la_SOURCES = \
 	src/own.hpp \
 	src/pair.cpp \
 	src/pair.hpp \
-	src/peer.cpp \
-	src/peer.hpp \
```

Comment:
please double check your rebase, these and other changes should not be in the commits - this is why the build is broken

Reply:
Hmmm... I'm pulling from libzmq/master, and those lines are not in my Makefile.am:

```
	src/own.hpp \
	src/pair.cpp \
	src/pair.hpp \
	src/pgm_receiver.cpp \
	src/pgm_receiver.hpp \
```

Maybe that's why the build fails?

FWIW, peer.cpp does appear in CMakelists.txt.

Reply:
That's why I suggested that the rebase was not successful, these lines you are removing need to be left there

Reply:
You're right -- not sure how that happened, I think it should be OK now.